src/psyclone/tests/psyir/nodes/schedule_test.py

```diff
@@ -51,9 +53,23 @@
     os.path.abspath(__file__)))), "test_files", "dynamo0p3")
 
 
+def test_sched_init():
+    ''' Check the Schedule class is initialised as expected.'''
```

Comment:
Seeing as we're keeping the children and parent optional arguments could you please add a test for each which checks that any provided value on initialisation is stored as expected in self.children and self.parent and that providing nothing results in self.children and self.parent being None.

Reply:
Done